docs/docs/connecting-to-eventstore/connecting-to-a-single-node.md

```diff
@@ -0,0 +1,44 @@
+# Connecting to a single node
+
+:::tip
+Packages needed:
+- `EventStore.Client.Grpc.Streams`   
+:::
+
+## Simple connection
+
+The simplest way to connect to EventStoreDB is to only use a single node and leave the rest of the options as default. 
+
+<<< @/samples/connecting-to-a-single-node/Program.cs#creating-simple-connection
+
+## Setting the connection name
+
+If you have multiple clients connecting to Event Store it is advisable to set the connection name so that it can be easily identified via the UI. This is done via a property on the `EventStoreClientSettings`.
```

Comment:
```suggestion
If you have multiple clients connecting to EventStoreDB it is advisable to set the connection name so that it can be easily identified via the UI. This is done via a property on the `EventStoreClientSettings`.
```